pkg/Directory.Build.props

```diff
@@ -7,6 +7,15 @@
     <IncludeSymbols>false</IncludeSymbols>
   </PropertyGroup>
 
+  <!-- nuspec properties -->
+  <PropertyGroup>
+    <Authors>Microsoft</Authors>
+    <PackageLicenseUrl>https://github.com/dotnet/machinelearning/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>http://dot.net/ml</PackageProjectUrl>
+    <PackageIconUrl></PackageIconUrl>
```

Comment:
I still need to get this URL from @shauheen.  My plan is to put a fwlink or aka.ms link here that points to the icon somewhere on the web.  For the .NET Core packages, we point our fwlink to Gravatar.